DQM/SiPixelMonitorCluster/src/SiPixelClusterSource.cc

```diff
@@ -344,6 +344,7 @@ void SiPixelClusterSource::buildStructure(const edm::EventSetup &iSetup) {
           thePixelStructure.insert(pair<uint32_t, SiPixelClusterModule *>(id, theModule));
         } else if (detId.subdetId() == static_cast<int>(PixelSubdetector::PixelEndcap)) {
           LogDebug("PixelDQM") << " ---> Adding Endcap Module " << detId.rawId() << endl;
+          /*
```

Comment:
This block is used to calculate `mask`, which is forced to `false`. If that code is not needed, we should delete it.

Reply:
Agreed, please delete the code.